### PR TITLE
Feature/ASC-331/ASC-597/ASC-635/view molecule orders/test cases

### PR DESCRIPTION
**Pull request by VamsiExcelra**
1. implemented test cases for 3 files related to molecule orders
2. send to synthesis button added and onclick triggering popup
3. added warning svg for failed case in molecule order
4. comments resolved
5. updated branch with main

### CODE REVIEW

File: apps/web/components/MoleculeOrder/MoleculeOrder.tsx [hunk 1]
```diff
@@ -220,25 +213,33 @@ const MoleculeOrderPage = ({ userData }: { userData: UserData }) => {
...
+            data={moleculeOrderData}
+            groupingColumn={rowGroupName()}
+            enableRowSelection
+            enableGrouping
+            enableInfiniteScroll={false}
+            enableSorting
+            enableFiltering={false}
+            enableOptions={false}
+            buttonText={buttonText} 
+            onButtonClick={openSynthesisPopup}
```
Comment (himanshubexcelra) [inline apps/web/components/MoleculeOrder/MoleculeOrder.tsx:240]: Here you are adding one button. Somewhere we are having multiple buttons like on molecule page. I asked you ro prepare one set of buttons and pass it to custom datagrid and then from there you should pass reference on click of button then based on reference you can open popup from your parent component

File: apps/web/app/molecule_order/page.tsx [hunk 2]
```diff
@@ -16,6 +25,7 @@ export default async function MoleculeOrder() {
 
   return (
     <Layout>
+      <Breadcrumb breadcrumbs={breadcrumbs} />
```
[the remark this answers is not in the document]
Comment (VamsiExcelra) [reply apps/web/app/molecule_order/page.tsx:28]: previously breadcrumb was loading in child molecule order component (use client) only but i have seen in the projects it is rendering in server side only - you can check 'projects - page.tsx' - after refering this i have updated my molecule order component breadcrumb too

[the remark this answers is not in the document]
Comment (VamsiExcelra) [reply apps/web/app/molecule_order/page.tsx:28]: as of now it is working fine without any issue

**Comment by himanshubexcelra** [comment]
Status column is not fixed.

File: apps/web/app/molecule_order/page.tsx (hunk 2 above)
[the remark this answers is not in the document]
Comment (himanshubexcelra) [reply apps/web/app/molecule_order/page.tsx:28]: you can move it. I will ask Forum to move it in other file. 
She might have missed it.

**Comment by himanshubexcelra** [comment]
image

Strcuture image is not coming.

File: apps/web/components/MoleculeOrder/MoleculeOrder.tsx (hunk 1 above)
Comment (VamsiExcelra) [reply apps/web/components/MoleculeOrder/MoleculeOrder.tsx:240]: toolbar Buttons logic implemented

**Comment by himanshubexcelra** [comment]
image


Typo error

**Comment by himanshubexcelra** [comment]
Display loader when data is loading from API. You can check other tables.

**Comment by himanshubexcelra** [comment]
Why we are using Unknown . Is it mentioned somewhere?

image

**Comment by himanshubexcelra** [comment]
Why did you change it to smile. Earlier it was smiles_string.

**Comment by himanshubexcelra** [comment]
Remove this console

image

**Comment by VamsiExcelra** [comment]
1.console removed
2.'unknown' is given while mapping data for grid - it is mandatory otherwise test case is failing because when there is no value is comes it handle it smoothly
3. loader added

**Pull request merged**